client/src/components/bonuses.jsx

```diff
@@ -13,24 +12,34 @@ class Bonuses extends React.Component {
       selectedSong: '', //string, representing url
     };
 
+    this.selectAlbum = this.selectAlbum.bind(this);
     this.selectSong = this.selectSong.bind(this);
   }
 
   componentDidMount() {
-    let bundleId = parseInt(window.location.pathname.slice(6));
-    console.log(bundleId)
+    let bundleId = window.location.pathname;
```

Comment:
Will you need to use `bundleId` outside of `componentDidMount`? If so, consider setting it as state within `componentDidMount` so that you don't lose access to the variable because of scoping.

Reply:
Yes! Thanks! I forgot to set the the state of `bundleNumber` to `bundleId`.